## Абсолютно непрерывное распределение. Плотность с индикатором

Рассмотрим случайную величину $$ \xi $$, имеющую плотность:

$$
p(x) = C \cdot x^{-3/2} \cdot \mathbb{I}_{[1, \infty)}
$$

где $$ \mathbb{I}_{[1, \infty)} $$ — индикатор множества: функция, равная 1 при $$ x \in [1, \infty) $$, и 0 в противном случае.

---

### Задача 1. Найти константу $$ C $$

Требование нормировки:

$$
\int_{-\infty}^{\infty} p(x) \, dx = 1
$$

Учитывая область действия индикатора:

$$
\int_1^{\infty} C \cdot x^{-3/2} \, dx = 1
$$

Вынесем $$ C $$ за знак интеграла:

$$
C \cdot \int_1^{\infty} x^{-3/2} \, dx = 1
$$

Решим интеграл:

$$
\int_1^{\infty} x^{-3/2} \, dx = \left[ \frac{x^{-1/2}}{-1/2} \right]_1^{\infty} = 2
$$

Отсюда:

$$
C = \frac{1}{2}
$$


In [6]:
from sympy import symbols, integrate, oo

x = symbols('x')
integral = integrate(x**(-3/2), (x, 1, oo))
C = 1 / integral
C.evalf()


0.500000000000000

### Задача 2. Найти вероятность

Вычислить:

$$
P\left( \frac{1}{10} < \frac{1}{\xi} < \frac{1}{5} \right)
$$

Перепишем неравенство через $$ \xi $$

$$
\frac{1}{10} < \frac{1}{\xi} < \frac{1}{5} \Rightarrow 5 < \xi < 10
$$

(так как $$ \xi > 0 $$, переворачиваем знаки)

Теперь вероятность:

$$
P(5 < \xi < 10) = \int_5^{10} p(x) \, dx = \int_5^{10} \frac{1}{2} \cdot x^{-3/2} \, dx
$$


In [15]:
# Вычисление вероятности: P(5 < ξ < 10)
prob = integrate(C * x**(-3/2), (x, 5, 10))
round(prob.evalf(), 4)


0.1310

### Ответы:

1. Нормировочная константа:
   $$ C = \frac{1}{2} $$

2. Вероятность:
   $$ P\left( \frac{1}{10} < \frac{1}{\xi} < \frac{1}{5} \right) = \boxed{0.1310} $$
